components/pedigree/resources/src/main/resources/pedigree/view/workspace.js

```diff
@@ -139,17 +140,17 @@ define([
             // set display:block
             svgText = svgText.replace(/(<svg[^<>]+style=")/g, "$1display:block; ");
```

Comment:
Good point. Unfortunately, there is a reason - the idea is to get a copy of the SVG, and leave the original as is. It is possible to modify DOM for the original, but then all of the things removed should be added back, all even handlers for removed parts restored, etc. This can be done, but it is very messy to re-run all the code which generated all of those things which need to be removed in the copy (@marta also think we should investigate this path, and I agree it is a good idea in general but it is not easy). This may also be visibly slow for a large pedigree (though this needs to be tested)

P.S. There are some things which are also hard to do in DOM due to browser bugs. E.g. when a gradient is used in an SVG, Chrome (and only Chrome) creates a <linearGradient> element for each gradient. And when an element is removed, that <linearGradient> is not removed (which is kind of ok in itself, but makes SVG messy). But even worse, if a copy of an SVG is used with the same <linearGradient>, and then the copy is removed, the gradients will break in the original SVG as well, even though original still has the <linearGradient> - though not sure what happens when they are removed and then redefined anew